client/my-sites/stats/stats-post-anniversaries/single-post.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Gridicon from 'gridicons';
+import { localize, moment } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import StatsContentText from '../stats-module/content-text';
+
+const periodLabel = ( translate, period ) => {
+	if ( period.period === 'day' ) {
+		return translate( 'this day' );
+	}
+	if ( period.period === 'week' ) {
+		return translate( 'this week' );
+	}
+	if ( period.period === 'month' ) {
+		return translate( 'this month' );
+	}
+	return '';
+};
+
+const SinglePost = localize( ( { translate, post, period } ) => {
+	const yearsAgo = period.startOf
+		.clone()
+		.startOf( 'year' )
+		.diff( moment( post.date ).startOf( 'year' ), 'years' );
+	return (
+		<StatsContentText>
+			<p>
+				<Gridicon icon="calendar" size={ 18 } />
+				{ translate(
+					'%(yearsAgo)d year ago on %(period)s, {{href}}%(title)s{{/href}} was published.',
+					'%(yearsAgo)d years ago on %(period)s, {{href}}%(title)s{{/href}} was published.',
```

Comment:
Concatenating translated strings (`%period`) like this can result in issues for some languages. Please rework to avoid inserting translated strings instead other translated strings.